src/rest/connector/libs/nd/implementation.py

```diff
@@ -185,7 +188,7 @@ def decorated(self, *args, **kwargs):
 
     @BaseConnection.locked
     @isconnected
-    def get(self, api_url, expected_status_code=requests.codes.ok,
+    def get(self, api_url, params=None, expected_status_code=requests.codes.ok,
             timeout=30, retries=3, retry_wait=10):
```

Comment:
May need to do it like this in the event anyone has used this method without specifying keywords
```suggestion
    def get(self, api_url, expected_status_code=requests.codes.ok,
            timeout=30, retries=3, retry_wait=10, params=None):
```